docs/getting-started/defining-models.md

```diff
@@ -0,0 +1,113 @@
+---
+id: defining-models
+title: Defining models
+---
+
+**Note:** If you're defining your own models, you also need to update the [collection configuration](Configuring-the-collection).
+
+
+## TypeScript
+
+**Note 🚨:** When using Babel 7 with the new decorators plugin, the [`@prop` decorators are ignored](https://github.com/infinum/datx/issues/92).
```

Comment:
I realised I forgot to remove this disclaimer when the issue was fixed in https://github.com/infinum/datx/commit/d681f69634ba0f5b15a3b9fac6629cbe01886771